README.md

```diff
@@ -1,67 +1,128 @@
-# Ruby Course
+# 📗 Table of Contents
 
-If you are not familiar with linters and GitHub Actions, read [root level README](../README.md).
+- [📖 About the Project](#about-project)
```

Comment:
- [ ] I am requesting these changes here because you have not changed the affected file in this PR. Take a look at your `person.rb` file.

  ![image](https://user-images.githubusercontent.com/36542986/211272470-3de59e4f-de87-496e-ac00-a58c88e7da16.png)
  
  - In the `initialize` method **`Random.ran(...)`** is not a valid Ruby expression so it throws errors. I think you meant this:
    ```
    Random.rand(...)
    ```

  - You need to define a method before trying to make it private.
    ```
    def of_age?
      @age >= 18
    end

    private :of_age?
    ```

  - In the `can_use_services?` method you need to use the instance variable `@parent_permission` and we need to make the code simple.

    ```
    def can_use_services?
      of_age? || @parent_permission
    end
    ```